Number_types/include/CGAL/Root_of_traits.h

```diff
@@ -76,7 +76,9 @@ compute_roots_of_2(const NT &a_, const NT &b_, const NT &c_, OutputIterator oit)
     }
   }
   else {
-    *oit++ = -c/b; return oit;
+      assert(false); 
+    // *oit++ = - c / b;  // AF: HELP 
```

Comment:
@sloriot can you uncomment and investigate.  If I write `NT(-c/b)`  it won''t work for `Quotient<MP_Float>` and without it has a problem with expression templates of boost::mp.

Reply:
It is strange that line 66 is not an issue then.